docs/docs/advanced/deploy-eliza-to-tee-with-oasis-rofl.md

```diff
@@ -0,0 +1,257 @@
+- Access control
+- Audit logging
+
+Here's a comprehensive list of secrets you might want to set for your Eliza agent:
+
+```bash
+# Basic Configuration
+# Set the character file path (you can use any character file of your choice)
+echo -n "characters/staff-engineer.character.json" | oasis rofl secret set CHARACTERS -
+echo -n "3000" | oasis rofl secret set SERVER_PORT -
+echo -n "5173" | oasis rofl secret set UI_PORT -
+echo -n "" | oasis rofl secret set UI_ALLOWED_HOSTS -
+echo -n "" | oasis rofl secret set UI_SERVER_URL -
+echo -n "" | oasis rofl secret set REMOTE_CHARACTER_URLS -
+echo -n "false" | oasis rofl secret set USE_CHARACTER_STORAGE -
+
+# Logging Configuration
+echo -n "log" | oasis rofl secret set DEFAULT_LOG_LEVEL -
+echo -n "false" | oasis rofl secret set LOG_JSON_FORMAT -
+echo -n "false" | oasis rofl secret set INSTRUMENTATION_ENABLED -
+echo -n "" | oasis rofl secret set EXPRESS_MAX_PAYLOAD -
+
+# OpenAI Configuration
+echo -n "sk-proj-XXXX" | oasis rofl secret set OPENAI_API_KEY -
+echo -n "" | oasis rofl secret set OPENAI_API_URL -
+echo -n "" | oasis rofl secret set SMALL_OPENAI_MODEL -
+echo -n "" | oasis rofl secret set MEDIUM_OPENAI_MODEL -
+echo -n "" | oasis rofl secret set LARGE_OPENAI_MODEL -
+echo -n "" | oasis rofl secret set EMBEDDING_OPENAI_MODEL -
+echo -n "" | oasis rofl secret set IMAGE_OPENAI_MODEL -
+echo -n "false" | oasis rofl secret set USE_OPENAI_EMBEDDING -
+
+# GitHub Integration
+echo -n "true" | oasis rofl secret set GITHUB_CLIENT_DISABLED -
+echo -n "true" | oasis rofl secret set GITHUB_PLUGIN_ENABLED -
+echo -n "ghp_XXXX" | oasis rofl secret set GITHUB_API_TOKEN -
+echo -n "5000" | oasis rofl secret set GITHUB_USER_CHECK_INTERVAL_MS -
+echo -n "5000" | oasis rofl secret set GITHUB_INFO_DISCOVERY_INTERVAL_MS -
+echo -n "20000" | oasis rofl secret set GITHUB_OODA_INTERVAL_MS -
+echo -n "10" | oasis rofl secret set GITHUB_ISSUES_LIMIT -
+echo -n "10" | oasis rofl secret set GITHUB_PULL_REQUESTS_LIMIT -
+
+# Database Configuration
+echo -n "postgresql://user:password@localhost:5432/eliza" | oasis rofl secret set POSTGRES_URL -
+```
+
+## Tutorial
+
+---
+
+### Prerequisites
+
+Before getting started with Eliza in TEE, ensure you have:
+
+- [Oasis CLI](https://github.com/oasisprotocol/cli/releases) installed
+- Docker Desktop or Orbstack (Orbstack is recommended)
+- Testnet tokens from the [Oasis Faucet](https://faucet.testnet.oasis.dev/)
+- A character file for your Eliza agent (you can use the example character file or create your own)
+
+---
+
+### Environment Setup
+
+To set up your environment for TEE development:
+
+1. **Create a keypair**
+
+    ```bash
+    oasis wallet create <account-name>
+    ```
+
+2. **Request Testnet Tokens**
+
+    Visit the [Oasis Faucet](https://faucet.testnet.oasis.dev/) to request TEST tokens.
+
+3. **Create a ROFL Application**
+
+    Create the ROFL application:
+
+    ```bash
+    oasis rofl create --network testnet --account <account-name>
```

Comment:
This will require an existing `rofl.yaml` in the project. Is it included? Otherwise one should run `oasis rofl init` in their project directory first.